### ISSUE TITLE
Socket.io blocked by print preview

### ISSUE THREAD

**Issue by kailniris** [nwjs/nw.js#4899]
Hello,
Well, as we know chrome print preview blocks all network traffic so socket.io will be disconnected after 60 sec (by default) if you stay in the print preview. Also it will block all incoming messages till you close the print dialog (atleast you will get the messages after you closed it expect if the socket timeout).

I know this is not your business cuz its a chrome implementation "bug" but i want to ask if is it possible somehow to create a new window which won't "block" the main window if i open the print dialog in it.

specs:
nw.js 1.4.5
win 10

**Comment by rogerwang** [comment]
This is the duplicate of https://bugs.chromium.org/p/chromium/issues/detail?id=178931

Opening a new window should help.

**Comment by kailniris** [comment]
Sorry I was not clear enough, here is a more accurate problem description:

The bellow code blocks the main window:
var w = window.open();
w.document.write(result);
w.print();
w.close()

The bellow code blocks the main window:
nw.Window.open('https://github.com/nwjs/nw.js', {}, function(new_win) {
  setTimeout(function() {
    new_win.window.print()
 }, 2000);
});

The bellow code **won't** block the main window:
nw.Window.open('https://github.com/nwjs/nw.js', {new_instance: true}, function(new_win) {
  //the problem is in here new_win will be undefined
});

The new_instance solution would be good but, i can't send data to this new window. I think this is related to a bug. Any suggestions?
